src/Domain/EarningsProfile.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace SFA.DAS.Funding.ApprenticeshipEarnings.Domain;
+
+public class EarningsProfile
+{
+    public decimal? AdjustedPrice { get; set; }
```

Comment:
why is it null-able?

Reply:
because it's calculated as part of the processing and starts off null